src/DynamoCore/Graph/Nodes/PortModel.cs

```diff
@@ -3,6 +3,7 @@
 using System.ComponentModel;
 using System.Linq;
 using System.Reflection;
+using System.Text.RegularExpressions;
```

Comment:
Why add this using without new code that uses it? I think you forgot to revert this change?

Reply:
Yes